owncloudData/src/main/java/com/owncloud/android/data/folderbackup/db/FolderBackupDao.kt

```diff
@@ -27,35 +28,40 @@ import com.owncloud.android.data.ProviderMeta
 import kotlinx.coroutines.flow.Flow
 
 @Dao
-abstract class FolderBackupDao {
-    @Query(
-        "SELECT * from " + ProviderMeta.ProviderTableMeta.FOLDER_BACKUP_TABLE_NAME + " WHERE " +
-                FolderBackUpEntity.folderBackUpEntityNameField + " = :name"
-    )
-    abstract fun getFolderBackUpConfigurationByName(
+interface FolderBackupDao {
+    @Query(SELECT)
+    fun getFolderBackUpConfigurationByName(
         name: String
     ): FolderBackUpEntity?
 
-    @Query(
-        "SELECT * from " + ProviderMeta.ProviderTableMeta.FOLDER_BACKUP_TABLE_NAME + " WHERE " +
-                FolderBackUpEntity.folderBackUpEntityNameField + " = :name"
-    )
-    abstract fun getFolderBackUpConfigurationByNameStream(
+    @Query(SELECT)
+    fun getFolderBackUpConfigurationByNameAsFlow(
         name: String
     ): Flow<FolderBackUpEntity?>
 
     @Insert(onConflict = OnConflictStrategy.REPLACE)
-    abstract fun insert(folderBackUpEntity: FolderBackUpEntity): Long
+    fun insertOrReplace(folderBackUpEntity: FolderBackUpEntity): Long
+
+    @Query(DELETE)
+    fun delete(name: String): Int
 
     @Transaction
-    open fun update(folderBackUpEntity: FolderBackUpEntity): Long {
+    fun update(folderBackUpEntity: FolderBackUpEntity): Long {
         delete(folderBackUpEntity.name)
-        return insert(folderBackUpEntity)
+        return insertOrReplace(folderBackUpEntity)
     }
 
-    @Query(
-        "DELETE from " + ProviderMeta.ProviderTableMeta.FOLDER_BACKUP_TABLE_NAME + " WHERE " +
-                FolderBackUpEntity.folderBackUpEntityNameField + " = :name"
-    )
-    abstract fun delete(name: String): Int
+    companion object {
+        private const val SELECT = """
```

Comment:
If we want to add query constants on every dao, we should align query names too. For example: `SELECT_WITH_NAME` or `DELETE_WITH_NAME`.

What do u think?

Reply:
Yes, I totally agree, but there are maaany queries and each of them have a different style... I think this could be work for a second iteration of renaming that we can do in a future 🤔

Reply:
Ok, let's do that on another round